src/views/portfolio/projects/ProjectDependencyGraph.vue

```diff
@@ -293,7 +302,7 @@ export default {
     },
     getDependencyUrl(dependency) {
       if (dependency.objectType === "COMPONENT") {
-        return `${this.$api.BASE_URL}/${this.$api.URL_COMPONENT}/${dependency.uuid}`;
+        return `${this.$api.BASE_URL}/${this.$api.URL_COMPONENT}/${dependency.uuid}?includeRepositoryMetaData=true`;
```

Comment:
`includeRepositoryMetaData` is hardcoded to `true`. It should use `fetchRepositoryMetaData` instead.

Reply:
`includeRepositoryMetaData` is meant to be always `true` in this case, because it allows to seemingly switch between highlighting all outdated components or not, regardless of the state of the toggle when expanding a node and fetching its children.
If fetching the metadata was dependent on `fetchRepositoryMetaData`, then only the components which were fetched while the toggle was set to active would be highlighted as oudated components, which would be very confusing for the user.

`fetchRepositoryMetaData` is definitely a very bad name for this variable, because its only purpose is to check if the information for the outdated components should be rendered or not.
It has actually nothing to do with fetching repository metadata, therefore I would change the name of it to `highlightOutdatedComponents`, but still keep the `includeRepositoryMetaData` in the URL hardcoded to `true`.

Reply:
Oh ok, got it. Makes sense. 👌

Agreed that changing the name would be good.